firebase_admin/_messaging_utils.py

```diff
@@ -153,11 +155,12 @@ class AndroidNotification(object):
         title_loc_args: A list of resource keys that will be used in place of the format specifiers
             in ``title_loc_key`` (optional).
         channel_id: channel_id of the notification (optional).
+        image: Image url of the notification (optional).
     """
 
     def __init__(self, title=None, body=None, icon=None, color=None, sound=None, tag=None,
                  click_action=None, body_loc_key=None, body_loc_args=None, title_loc_key=None,
-                 title_loc_args=None, channel_id=None):
+                 title_loc_args=None, channel_id=None, image=None):
```

Comment:
Need the same added to `APNSFCMOptions`.  See https://github.com/firebase/firebase-admin-java/pull/298